src/commands/rspec.yml

```diff
@@ -29,7 +29,8 @@ parameters:
     default: ''
 
 steps:
-  - browser-tools/install-chrome
+  - browser-tools/install-chrome:
+      chrome-version: 114.0.5735.198
```

Comment:
The latest version of chrome, 115.0.5790, cannot be installed, so 114.0.5735.198 must be specified.